master/docs/relnotes/index.rst

```diff
@@ -72,6 +72,8 @@ Deprecations, Removals, and Non-Compatible Changes
 
 * Master/worker protocol has been changed:
 
+  * ``slave_commands`` key in worker information was renamed to ``worker_commands``.
```

Comment:
usually, you create separate PR for relnotes, especially when there is a PR burst like this.